docker/nginx-ui-local-dev-only.conf

```diff
@@ -0,0 +1,49 @@
+# This file overrides ui/web-ui-nginx-site.conf
+# for the local docker compose usage.
+# It is a simplified version and doesn't include stricter security headers.
+# It is to be used in production!
```

Comment:
"It is *not* to be used in production" I guess

Reply:
Good catch :)

Reply:
lol.. my bad
Thanks :)